### DISCUSSION TITLE
twitter blogs not limiting threads

### DISCUSSION THREAD

**Discussion by fusionmc-evilscoop** [TumblThreeApp/TumblThree#455] [General]
Getting a lot of the following in the logs -

You should lower the connections to the tumblr api in the Settings->Connection pane., The remote server returned an error: (429) Too Many Requests.

These are ALL twitter api blogs, so the connections to tumblr api do not apply
While the recent fix means that twitter blogs are now online again (thx), they wont actually download due to threads hammering and twitter blocking

**Comment by thomas694** [comment]
Of course the download of the blogs works too!
But since the platform has "evolved" a lot over the past few weeks...
- some changed data structures
  Most of the blogs work again, but not all especially if they have special tweets in it. Everyone can help here by sending this issue number together with the failing blog url (and tweet id/date if possible) via the feedback dialog or by email.
- very low rate limits
  The current rate limit is around 5 requests/min, so depending on your configured settings you download more or less sawtooth-shaped. But anyway, the download continues automatically after the waiting period ("waiting until hh:mm:ss").
- no separate rate limiter configuration yet
  Currently the rate limiter is shared. Depending on whether you have blogs from both platforms or not, you can also run a second instance of the app for configuring different values.

...
